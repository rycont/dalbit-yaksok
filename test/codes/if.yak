만약 4 > 3 이면 # 주석
    "4는 3보다 크지" 보여주기
만약 1 > 2 이면 # 주석
    "1은 2보다 크지" 보여주기
