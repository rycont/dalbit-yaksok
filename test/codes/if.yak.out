4는 3보다 크지
